src/pages/buyer-signup/buyer-signup.html

```diff
@@ -0,0 +1,79 @@
+<!--
+  Generated template for the BuyerSignupPage page.
+
+  See http://ionicframework.com/docs/components/#navigation for more info on
+  Ionic pages and navigation.
+-->
+<ion-header style="background-color: #F44336;">
+
+  <ion-navbar>
+ 
+          <ion-icon ios="ios-arrow-round-back" md="md-arrow-round-back" slot="start" style="color: #fff"></ion-icon>
+        <div class="container">
+          <!-- <ion-title>buyer-signup-page</ion-title> -->
+          <img src="../../assets/imgs/sourcing produce logosource.svg">
```

Comment:
please next time this can be reduced to  assets/logo/logo.svg

Reply:
I'll resolve it

Reply:
I'll resolve it